README.md

```diff
@@ -173,6 +173,23 @@ Your email will be acknowledged within 24 hours, and you’ll receive a more
 detailed response to your email within 48 hours indicating the next steps in
 handling your report.
 
+There is not any hard and fast rule to determine if a bug is worth reporting.
```

Comment:
Nit: Consider changing `worth reporting` here and below to something else? If it's a bug, it's worth reporting. It's just a question of whether it's a security issue or whether the bug is in Node.js itself or end-user code.

Maybe change this entire sentence to:

> It can be difficult to determine if an issue is a valid security issue or not.

Then change `any attack worth reporting via the security address` to `a valid security issue`?